go-fuzz/main.go

```diff
@@ -40,8 +48,16 @@ func main() {
 	if *flagMaster != "" && *flagSlave != "" {
 		log.Fatalf("both -master and -slave are specified")
 	}
+	if *flagMasterStats != "" && *flagSlave != "" {
+		log.Fatalf("both -masterstats and -slave are specified")
+	}
 	if *flagPprof != "" {
-		go http.ListenAndServe(*flagPprof, nil)
+		go func() {
+			err := http.ListenAndServe(*flagPprof, nil)
```

Comment:
panic(http.ListenAndServe(*flagPprof, nil))
ListenAndServe does not return with nil error